script.js

```diff
@@ -1,4 +1,310 @@
+      ", you played " +
+      input +
+      " and the computer played " +
+      computerTurn +
+      " so you win. <br>" +
+      yourname +
+      " Wins:" +
+      userWins +
+      "(" +
+      ((userWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")" +
+      "<br> Computer Wins: " +
+      compWins +
+      "(" +
+      ((compWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")";
+    // Return output.
+    console.log("you played ", input, " and computer played ", computerTurn);
+    return myOutputValue;
+  }
+
+  // Paper vs paper (draw)
+  if (
+    (input == "paper" || input == "reversed paper") &&
+    computerTurn == "paper"
+  ) {
+    myOutputValue =
+      yourname +
+      " you played " +
+      input +
+      " and the computer played " +
+      computerTurn +
+      " so you drew. <br>" +
+      yourname +
+      " Wins:" +
+      userWins +
+      "(" +
+      ((userWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")" +
+      "<br> Computer Wins: " +
+      compWins +
+      "(" +
+      ((compWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")";
+    // Return output.
+    console.log("you played ", input, " and computer played ", computerTurn);
+    return myOutputValue;
+  }
+
+  // Scissors
+  // Scissors vs Scissors (draw)
+  if (
+    (input == "scissors" || input == "reversed scissors") &&
+    computerTurn == "scissors"
+  ) {
+    myOutputValue =
+      yourname +
+      " you played " +
+      input +
+      " and the computer played " +
+      computerTurn +
+      " so you drew. <br>" +
+      yourname +
+      " Wins:" +
+      userWins +
+      "(" +
+      ((userWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")" +
+      "<br> Computer Wins: " +
+      compWins +
+      "(" +
+      ((compWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")";
+    // Return output.
+    console.log("you played ", input, " and computer played ", computerTurn);
+    return myOutputValue;
+  }
+
+  // Scissors vs paper (win) or reversed scissors vs stone (win)
+  if (
+    (input == "scissors" && computerTurn == "paper") ||
+    (input == "reversed scissors" && computerTurn == "stone")
+  ) {
+    userWins = userWins + 1;
+    myOutputValue =
+      yourname +
+      ", you played " +
+      input +
+      " and the computer played " +
+      computerTurn +
+      " so you win. <br>" +
+      yourname +
+      " Wins:" +
+      userWins +
+      "(" +
+      ((userWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")" +
+      "<br> Computer Wins: " +
+      compWins +
+      "(" +
+      ((compWins / (draws + compWins + userWins)) * 100).toFixed(2) +
+      "%" +
+      ")";
+    // Return output.
+    console.log("you played ", input, " and computer played ", computerTurn);
+    return myOutputValue;
+  }
+};
+
```

Comment:
Consider grouping all the win conditions into one if / else if condition, same for all the lose and draw conditions, so as to minimise repetition in code for the variable myOutputValue